admin/src/app/views/usuarios/usuarios.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="c-usuarios">
+  <button type="button" class="c-usuarios_boton-nuevo btn btn-success" (click)="abrirModalNuevo()">Nuevo</button>
```

Comment:
typo, no se aplica el estilo
```suggestion
  <button type="button" class="c-usuarios__boton-nuevo btn btn-success" (click)="abrirModalNuevo()">Nuevo</button>
```